CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/editors/DateEditorFactory.java

```diff
@@ -71,7 +71,7 @@ protected LocalDate getValueFromControls() throws ControlsInvalidException {
             }
             try {
                 final String dateString = datePicker.getEditor().getText();
-                final LocalDate date = datePicker.getConverter().fromString(dateString);
+                datePicker.getConverter().fromString(dateString);
```

Comment:
Has there been checks done on what these functions actually do? Some functions you can (sort of) get away with not assigning to a variable but a lot don't actually do anything really without the assignment (and in those circumstances it may be better to just remove the line all together).

Reply:
Yes, I left that one there because the function is being used to test if the dateString was inputted in the correct format. So if it can't be converted from the string, its not in the correct format and will throw an exception. It works but the converter isn't being used for what its designed for.

Reply:
Alright sweet. I think there are a few others scattered around the place. Can I get confirmation that they fall in the same category?

Reply:
Did you want me to go through and comment on these ones and explain why I left them?

Reply:
It probably wouldn't hurt to leave a comment. Just so people in the future know that something is happening in the background